pyvista/core/filters/data_set.py

```diff
@@ -935,7 +935,7 @@ def threshold(
         scalars=None,
         invert=False,
         continuous=False,
-        preference='cell',
+        preference='point',
```

Comment:
Noting https://github.com/pyvista/pyvista/pull/3744/files/139e0be1c1bba49cffa28e9d9db6b8e5343ca651#r1056828388

This change is likely a breaking change for many users. We may want to handle this change in the `threshold()` operations in one PR when updating the filter for #3610

Reply:
Reverted to deal with later